### ISSUE TITLE
mAP graph

### ISSUE THREAD

**Issue by rrrtype** [AlexeyAB/darknet#2334]
Because I am doing a limited object object, it's more than 99% mAP.
So, graph of mAP in chart.png is very hard to see.
Is there a way to improve it?

**Comment by AlexeyAB** [comment]
Can you show screenshot?

**Comment by AlexeyAB** [comment]
@kocica Yes, you can place mAP together with TP/FP/FN into one graph, because mAP@0.5 is a common accuracy parameter for object detection competitions (Pascal VOC, ImageNet, MS COCO, ...) and for comparisons, but TP/FP/FN is easier to understand.

**Comment by wonchulSon** [comment]
@rrrtype Hi, bro
How did you get the mAP?

**Comment by AlexeyAB** [comment]
@wonchulSon Just use this repository https://github.com/AlexeyAB/darknet and add flag `-map` at the end of training command.
